components/match2/wikis/arenaofvalor/match_group_input_custom.lua

```diff
@@ -185,12 +202,18 @@ function CustomMatchGroupInput.getResultTypeAndWinner(data, indexedScores)
 	return data, indexedScores
 end
 
+---@param opponents table[]
+---@param winner nil
```

Comment:
annos for winner, specialType, finished seem wrong
like there is stuff checking them for some values, hence always having them be nil makes no sense to me